Tasks/Maven/pmdformaven.ts

```diff
@@ -10,25 +10,27 @@ import trm = require('vsts-task-lib/toolrunner');
 
 import ar = require('./analysisresult');
 
+export const toolName:string = 'PMD';
+
 // Adds PMD goals, if selected by the user
 export function applyPmdArgs(mvnRun: trm.ToolRunner):void {
-    mvnRun.arg(['jxr:jxr', 'pmd:pmd']); // JXR source code cross-referencing is a required pre-goal for PMD
```

Comment:
As discussed offline, there doesn't seem to be any advantage to adding _-DlinkXRef=false_ unless the omitting it results in a warning/error.